src/pages/ExamsPage/components/ExternalReviewDashboard.jsx

```diff
@@ -1,13 +1,35 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
+import { Button } from '@edx/paragon';
+import { useIntl } from '@edx/frontend-platform/i18n';
 import { getLaunchUrlByExamId } from '../utils';
+import messages from '../messages';
 
-const ExternalReviewDashboard = ({ exam }) => (
-  <div data-testid="review_dash">
-    {exam && <iframe title="lti_tool" src={getLaunchUrlByExamId(exam.id)} width="100%" height="1100" style={{ border: 'none' }} />}
-  </div>
-);
+const ltiToolEmbed = false; // This is set to false for now but will eventually need to be a configurable variable.
+
+const ExternalReviewDashboard = ({ exam }) => {
+  const { formatMessage } = useIntl();
+
+  // By default, launch instructor tool in a new tab. Otherwise, embed in dashboard as iframe.
+  return (
+    <div data-testid="review_dash">
+      <div>
+        {!ltiToolEmbed && exam && (
+          // todo: add spacing
```

Comment:
Right now, there is no spacing/padding around the button (see screenshot). I tried by adding `classname='p-0'` to the button based on similar examples, but I'm now wondering whether this spacing should be added to the component above (i.e. exams page or just the review dashboard)

Reply:
I'd look at paragon for layout options and/or a containing element for the button. Maybe https://paragon-openedx.netlify.app/components/actionrow/?

Reply:
Added some padding.